lib/tasks/apipie.rake

```diff
@@ -195,10 +195,15 @@ namespace :apipie do
   end
 
   def generate_swagger_using_args(args, out)
-    args.with_defaults(:version => Apipie.configuration.default_version,
-                       :swagger_content_type_input => Apipie.configuration.swagger_content_type_input || :form_data,
-                       :filename_suffix => nil)
-    Apipie.configuration.swagger_content_type_input = args[:swagger_content_type_input].to_sym
+    args.with_defaults(
+      version: Apipie.configuration.default_version,
+      content_type_input: Apipie.configuration.generator.swagger.content_type_input,
+      filename_suffix: nil
+    )
+
+    Apipie.configuration.generator.swagger.content_type_input =
+      args[:swagger_content_type_input].to_sym
```

Comment:
I am a bit confused how above, you've renamed `swagger_content_type_input` in the defaults as `content_type_input` but then, you keep using `swagger_content_type_input` here